railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -284,6 +287,14 @@ def initialize(*args)
           self.options = options.merge(skip_sprockets: true, skip_javascript: true).freeze
         end
 
+        # Skip yarn for minimal apps, actionmailer, actioncable, webpacker, activestorage
```

Comment:
This comment should probably be reworded... or we can just remove it since `if options[:minimal]` is fairly self-documenting.